asm/istio/istio-operator.yaml

```diff
@@ -17,7 +17,7 @@ kind: IstioOperator
 spec:
   profile: asm-gcp
   hub: gcr.io/gke-release/asm # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.hub"}
-  tag: 1.7.3-asm.6 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.tag"}
+  tag: 1.8.1-asm.5 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.tag"}
```

Comment:
we'd better use `kpt set asm anthos.servicemesh.tag 1.8.1-asm.5` to also update Kptfile:115.

Reply:
Updated in the jump server script.